server/modules/authentication/discord/authentication.js

```diff
@@ -6,9 +6,22 @@
 
 const DiscordStrategy = require('passport-discord').Strategy
 const _ = require('lodash')
+const DiscordOauth2 = require('./node_modules/discord-oauth2/index.js') //I don't remember why I 
```

Comment:
I would remove the comment and make the `DiscordOauth2` import above lodash 👍 

```suggestion
const DiscordOauth2 = require('./node_modules/discord-oauth2/index.js')
const _ = require('lodash')
```